src/Avalonia.Visuals/Media/PathFigure.cs

```diff
@@ -25,6 +28,12 @@ public sealed class PathFigure : AvaloniaObject
         public static readonly StyledProperty<Point> StartPointProperty
                         = AvaloniaProperty.Register<PathFigure, Point>(nameof(StartPoint));
 
+        internal event EventHandler SegmentsInvalidated;
+
+        private IDisposable? _segmentsObserver;
```

Comment:
The `?` here and on the next line is producing a warning because nullable reference types aren't enabled for this file. Either add `#nullable enable` or remove the `?`.